src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -4,13 +4,16 @@
 import uk.gov.hmcts.cmc.claimstore.models.Claim;
 import uk.gov.hmcts.cmc.claimstore.models.amount.AmountBreakDown;
 import uk.gov.hmcts.cmc.claimstore.models.otherparty.TheirDetails;
+import uk.gov.hmcts.cmc.claimstore.utils.Formatting;
 
 import static java.util.Objects.requireNonNull;
 import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatMoney;
 
 public class CCJContent {
 
     private String claimReferenceNumber;
+    private String claimantName;
+    private String ccjRequestedDate;
```

Comment:
No need to prefix it with CCJ since this content is all about CCJ.

Reply:
not a major thing, will change the name